query-engine/driver-adapters/src/napi/conversion.rs

```diff
@@ -12,6 +12,12 @@ impl FromNapiValue for JSArg {
     }
 }
 
+impl FromNapiValue for JSArgType {
+    unsafe fn from_napi_value(_env: napi::sys::napi_env, _napi_value: napi::sys::napi_value) -> napi::Result<Self> {
```

Comment:
Why do we need to implement this trait?

Reply:
It doesn't compile otherwise, you may see there are other `FromNapiValue` implementations in the same module for the same reasons